catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -407,18 +418,21 @@
                             javax.xml.transform.sax,
                             javax.xml.transform.stream,
                             javax.xml.xpath,
+                            org.apache.commons.collections,
```

Comment:
I don't see the `commons-collections` dependency in this jar. And, this one should be an embed once the dependency is added.